RestrictEvents/RestrictEvents.cpp

```diff
@@ -263,6 +260,55 @@ struct RestrictEventsPolicy {
 		return true;
 	}
 
+	static void getBlockedProcesses(BaseDeviceInfo *info) {
+		// Updates procBlacklist with list of processes to block
+		char duip[128] { "auto" };
+		if (PE_parse_boot_argn("revblock", duip, sizeof(duip))) {
+			DBGLOG("rev", "read revblock from boot-args");
+		} else if (readNvramVariable(NVRAM_PREFIX(LILU_VENDOR_GUID, "revblock"), u"revblock", &EfiRuntimeServices::LiluVendorGuid, duip, sizeof(duip))) {
+			DBGLOG("rev", "read revblock from NVRAM");
+		}
+
+		char *value = reinterpret_cast<char *>(&duip[0]);
+		value[sizeof(duip) - 1] = '\0';
```

Comment:
Isn’t `sizeof(duip) - 1` always 127? Rather `strlen`?

Reply:
This is re-used code from elsewhere